### ISSUE TITLE
Face Verification Model

### ISSUE THREAD

**Issue by binguliki** [abhisheks008/DL-Simplified#512]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Face Verification Model Using Siamese Neural Network</br>
:red_circle: **Aim** :  To develop a Siamese neural network trained over triplet loss function for face verification task</br>
:red_circle: **Dataset** :  [LFW dataset](https://vis-www.cs.umass.edu/lfw/)</br>
:red_circle: **Approach** : Use LFW dataset to generate triplets for training the model and develop a siamese neural network architecture which would be used in Facial verification task

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by binguliki** [comment]
Please assign this issue to me as i'm currently working on this project and also do provide the GSSoC24 tag for my contribution to be counted.

Full name : Bingumalla Likith
GitHub Profile Link : [binguliki](https://github.com/binguliki)
Email ID : blikith86@gmail.com
Participant ID (if applicable): NA
Approach for this Project : I would be Using a transfer learning and structure my Siamese model for better results 
What is your participant role? GSSoC24 | Contributor

**Comment by binguliki** [comment]
@abhisheks008 Hey 👋 Bro , Can you please check this out !!

**Comment by abhisheks008** [comment]
Can you use at least 2-3 deep learning models for this dataset, and compare them on the accuracy scores to find out the best fitted model for this dataset?

**Comment by binguliki** [comment]
@abhisheks008 Sure bro i'll compare different architectures like InceptionResnet , VGG19 , Mobilenet etc perfomances and give the best results👍🏻 . I eagerly want to contribute and become a part of the community .
Can you assign me the Project

**Comment by abhisheks008** [comment]
> Siamese

Don't forget to add Siamese.

Issue assigned to you @binguliki

**Comment by utsabsamadder** [comment]
@abhisheks008 I would like to work on this as well. I propose to use YOLO which is much faster than Neural Network and for give a better result collectively.

**Comment by abhisheks008** [comment]
As per the code of conduct of the repository, only one contributor can work on one issue at a time.
